csharp/test/Drivers/Snowflake/ClientTests.cs

```diff
@@ -151,7 +151,7 @@ public void CanClientExecuteQueryUsingPrivateKey()
         {
             SnowflakeTestConfiguration testConfiguration = Utils.LoadTestConfiguration<SnowflakeTestConfiguration>(SnowflakeTestingUtils.SNOWFLAKE_TEST_CONFIG_VARIABLE);
 
```

Comment:
This will fail if the configuration is not using JWT. Here is a suggested change:

``` 
 Skip.If(testConfiguration.Authentication.SnowflakeJwt is null, "JWT authentication is not configured");
```